src/components/carousels/main-carousel/MainCarousel.css

```diff
@@ -0,0 +1,113 @@
+.carousel-container {
+  width: 100%;
+  aspect-ratio: 64/25;
+  z-index: -1 !important;
+  margin: 50px 0 50px 0;
+}
+.carousel-inner {
+  width: 100%;
+  height: 100%;
+}
+.slide-wrapper {
+  width: 100%;
+  aspect-ratio: 64/22;
+}
+.carousel-card {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  bottom: 10%;
+  opacity: 0;
+  pointer-events: none;
+  transform: translate(100%, 0);
+  transition: 1s ease-in-out;
+}
+.carousel-card:hover {
+  flex: 1;
+  position: absolute;
+  height: 100%;
+  overflow: hidden;  
+}
+.carousel-card-active {
+  opacity: 1;
+  /* transform: translate(-100%); */
```

Comment:
There are a couple of things that have been commented out. Would you like to keep them that way for now in case you need them later or can they just be removed?

Reply:
Will review :)